cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<p>I might put this specific example later, because this is not the recommended approach-- it would be easier/better just to re-balance &amp; overwrite the divisive weights.</p><p> </p><p>Also this title can be changed to something more specifici- e.g. How to modify correction weights stored in bins</p><p> </p><p> </p><p>Also, the current example is a little confusing, as it takes divisive weights, makes them multiplicative &amp; then discards them...</p><p> </p><p> </p><p>A nice example might be to take the weights of one cooler &amp; apply them to another... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I think sometimes people want to ensure that they use the same weights for visualization in juicebox and analysis with cooltools, for example, or to compare analyses of the two... And it's certainly faster than rebalancing, which might make a difference sometimes.</p><p> </p><p>I just wanted to show how to remove a column, as an example... Idk I think it might be useful sometimes? Also tbh I just wanted the file to remain in the original state just in case, so it doesn't make something in some other notebook look weird.</p>

Reply:
two concepts to introduce:
- how mcool resolutions are stored in hdf5 (from cooler docs, which could be. linked?):  
The current standard for Hi-C coolers is to name multi-resolution coolers under .mcool extension, and store differrent resolutions in an HDF5 group resolutions, as shown above.)

could modify code so that the resolution grabbed from the hdf5 file is the same as the cooler being considered elsewhere (e.g.  use `resolution = clr.bin-size` which is then passed as a key to `f`)